python/torch_mlir_e2e_test/test_suite/elementwise.py

```diff
@@ -1130,6 +1130,51 @@ def ElementwiseMishModule_basic(module, tu: TestUtils):
     module.forward(tu.rand(5, 3, low=-3.0, high=3.0))
 
 
+# ==============================================================================
+
+class ElementwiseXlogyTensorModule(torch.nn.Module):
+
+    def __init__(self):
+        super().__init__()
+
+    @export
+    @annotate_args([
+        None,
+        ([-1, -1], torch.float32, True),
+        ([-1, -1], torch.float32, True),
+    ])
+    def forward(self, x, y):
+        return torch.ops.aten.xlogy(x, y)
+
+
+@register_test_case(module_factory=lambda: ElementwiseXlogyTensorModule())
+def ElementwiseXlogyTensorModule_basic(module, tu: TestUtils):
+    module.forward(tu.rand(3, 5), tu.rand(3, 5))
```

Comment:
Can you also add tests that explicitly test the zero and NAN case? (i.e. doing `module.forward(torch.zeros(3, 5), ...)` and `module.forward(... , torch.tensor(float('nan')))`